modules/nf-core/toulligqc/tests/main.nf.test

```diff
@@ -0,0 +1,130 @@
+nextflow_process {
+
+    name "Test Process TOULLIGQC"
+    script "../main.nf"
+    process "TOULLIGQC"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "toulligqc"
+
+    test("sarscov2 - nanopore sequencing_summary") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['nanopore']['test2_sequencing_summary'], checkIfExists: true)
+                    ]
+                input[1] = [
+                    [],
+                    []
+                    ]
+                input[2] = [
+                    [],
+                    []
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.report_data.get(0).get(1)).readLines()[11..74]).match() },
+                { assert process.out.report_html[0] ==~ ".*/report.html"}
+            )
+        }
+
+    }
+
+    test("sarscov2 - nanopore fastq") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [],
+                    []
+                    ]
+                input[1] = [
+                    [ id:'test' ], // meta map
+                    file(params.test_data['sarscov2']['nanopore']['test_fastq_gz'], checkIfExists: true)
+                    ]
+                input[2] = [
+                    [],
+                    []
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(file(process.out.report_data.get(0).get(1)).readLines()[11..67]).match() },
+                { assert process.out.report_html[0] ==~ ".*/report.html"}
+            )
+        }
+
+    }
+
+    test("sarscov2 - nanopore bam") {
```

Comment:
Can you add one test with all three if they are not mutually exclusive

Reply:
I can do but I will need to upload new data to the nf-core/tets-datasets repo as I don't think the sequencing summary, FASTQ and BAM files come from the same run..

Reply:
Ah ok, that would be a bit awkward... maybe you could do a local test and otherwise I avert my eyes ;)

Reply:
Okay thank you! I'll do a local test this afternoon and will let you know the outcome :)

Reply:
I have done a test with a sequencing summary and fastq file and it worked!
![local_test_seq_summary_fastq](https://github.com/nf-core/modules/assets/91611978/eb855e4e-b9d3-45b9-ab8f-d1942ddf7b68)